Makefile

```diff
@@ -26,12 +26,13 @@ endif
 
 
 build: sbt
-	cd $(RESTSRV_DIR); \
+	cd $(RESTSRV_DIR) && \
 		$(SBT) ++$(SCALA_VERSION) -batch server/universal:stage
 
 clean: sbt
-	cd $(RESTSRV_DIR); \
+	cd $(RESTSRV_DIR) && \
 		$(SBT) ++$(SCALA_VERSION) -batch server/clean
+	find $(RESTSRV_DIR) -name "__pycache__" -exec rm -r {} \; || /bin/true
```

Comment:
Doesn't bash have `true`? because I don't have `/bin/true` and have always used `true` in cases like this. From reading, `/bin/true` is linux/gnu specific

Reply:
As I'm aware `/bin/true` is *Nix specific.

Reply:
and yet I don't have it. Does `true` not work someplace?